qosc/OscSender.cpp

```diff
@@ -71,13 +71,14 @@ void OscSender::send(const QString& oscAddress, const QVariantList& arguments) {
 }
 
 
-void OscSender::variantListToByteArray(QByteArray& outputResult, const QString& oscAddress, const QVariantList& arguments) {
+void OscSender::variantListToByteArray(QByteArray& outputResult, const QVariantList& arguments) {
     char buffer[1024];
     osc::OutboundPacketStream packet(buffer, 1024);
     // FIXME: Sending datagrams larger than 512 bytes is in general disadvised, as even if they are sent successfully,
     // they are likely to be fragmented by the IP layer before arriving at their final destination.
     // packet << osc::BeginBundleImmediate << osc::BeginMessage(oscAddress.toStdString().c_str());
-    packet << osc::BeginMessage(oscAddress.toStdString().c_str());
+    QString loadString = "";
+    packet << osc::BeginMessage(loadString.toStdString().c_str());
```

Comment:
for some reason, this code now makes the first argument of the received OSC packet an empty string, which complicates direct parsing in Max/MSP (the address is now 2nd, and so forth).
![Screenshot_2](https://user-images.githubusercontent.com/34627009/133358048-611f998d-79ed-4dc5-b264-e7d9ce977eb3.png)
this could be fixed by reverting back to having the OSC address as its own function argument, or further refactoring so that the new implementation is maintained, but replacing `oscAddress` with `arguments[0]` and excluding index 0 in the subsequent argument appends. i'm guessing there was a reason for doing this in the first place so i'll leave it to your better judgement :+)